packages/multichain-network-controller/CHANGELOG.md

```diff
@@ -7,4 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-[Unreleased]: https://github.com/MetaMask/core/
+## [0.1.0]
+
+### Added
+
+- Initial release ([#5215](https://github.com/MetaMask/core/pull/5215))
+  - Handle both EVM and non-EVM network and account switching for the associated network.
+  - Added `NetworkControllerSetActiveNetworkAction`
```

Comment:
Nit: do we need this here ?
```suggestion
NetworkControllerSetActiveNetworkAction

Reply:
Nope, not really (I missed that one, thanks 😄)